src/sprites.asm

```diff
@@ -844,88 +1250,275 @@ BEQ $03
 JMP KillWithStar
 LDA #$00
 JSR LocateContact
-BEQ .koopaWins
-LDA #$02
-JSR BounceSquash
+BEQ NetKoopaWin
+LDA $0DB9
+AND #$01
+BNE NetKoopaSpinKill
 LDA #$00
 STA $00AA,y
 STA $00B6,y
+LDA #$02
+JSR BounceSquash
+RTS
+.return
+RTS
+NetKoopaSpinKill:
+JSR SpinKillSprite
+RTS
+NetKoopaWin:
+JSR HurtLuigi
+RTS
+
+
+;thwonp + thwimp = jumping piranah
+
+Sprite28:;Big Boo
 RTS
 
-.koopaWins
+Sprite29:;Koopaling
+LDA $1570,x
+BEQ $03
+JMP KillWithStar
+LDA #$00
+JSR LocateContact;\if the chuck wins
+BEQ KoopalingWins;/hurt player
+JSR BoostSpeed
+LDA #$02;\bounce sound
+STA $1DF9;/
+JSR Sprite1C
+JSR Sprite4A
+LDA $00C2,y;\if stunned
+INC; |
+STA $1528,y;/
+CMP #$01;\if three, kill
+BEQ KillKoopaling;/
+
+LDA #$28                ;\ Play sound effect ;From chuck dissasembly
+STA $1DFC               ;/
+LDA #$00
+JSR KillKoopaling
+PHY
+LDA KoopalingBounceXsp,Y
+PLY
+STA $00B6,x
+RTS
+
+KillKoopaling:
+LDA #$02
+STA $14D1,y
+KoopalingBounce:
+RTS
+
+KoopalingWins:
 JSR HurtLuigi
-.return
 RTS
 
-			;thwonp + thwimp = jumping piranah
+KoopalingBounceXsp:
+db $20,$E0
+RTS
+
+Sprite2A:;upside down pirannah plant
+JSR Sprite4F
+RTS
 
-Sprite28:
+Sprite2B:;Sumo Brother Ligth
+JSR Sprite4F
 RTS
 
-Sprite29:
+Sprite2C:;Yoshi egg
 RTS
 
-Sprite2A:		;upside down piranah plant
+Sprite2D:;Baby Green Yoshi
```

Comment:
There's no point in supporting the baby yoshi since it's impossible to support adult yoshi. There just isn't enough space in SP1 VRAM for both luigi and yoshi.

Reply:
I know, but I could not leave it as it was :v